packages/d3fc-webgl/src/series/glOhlc.js

```diff
@@ -0,0 +1,245 @@
+import attributeBuilder from '../buffers/attributeBuilder';
+import uniformBuilder from '../buffers/uniformBuilder';
+import glScaleBase from '../scale/glScaleBase';
+import programBuilder from '../program/programBuilder';
+import ohlcShader from '../shaders/ohlc/shader';
+import drawModes from '../program/drawModes';
+import { rebind } from '@d3fc/d3fc-rebind';
+
+export default () => {
+  let program = programBuilder();
+  let xScale = glScaleBase();
+  let yScale = glScaleBase();
+  let decorate = () => {};
+  let xValues = null;
+  let open = null;
+  let high = null;
+  let low = null;
+  let close = null;
+  let bandwidth = null;
+  let lineWidth = null;
+
+  const xValueAttrib = 'aXValue';
+  const yValueAttrib = 'aYValue';
+  const xLineWidthAttrib = 'aXLineWidth';
+  const yLineWidthAttrib = 'aYLineWidth';
+  const bandwidthAttrib = 'aBandwidth';
+  const colorAttrib = 'aColor';
+
+  const draw = (numElements) => {
+    const shaderBuilder = ohlcShader();
+    program.vertexShader(shaderBuilder.vertex())
+      .fragmentShader(shaderBuilder.fragment())
+      .mode(drawModes.TRIANGLES);
+
+    xScale.coordinate(0);
+    xScale(program);
+    yScale.coordinate(1);
+    yScale(program);
+
+    program.buffers().uniform('uScreen', uniformBuilder([
+      program.context().canvas.width,
+      program.context().canvas.height
+    ]));
+
+    program.vertexShader()
+      .appendBody(`
+        gl_Position.x += ((aXLineWidth / 2.0) + (aBandwidth / 2.0)) / uScreen.x;
+        gl_Position.y += (aYLineWidth / 2.0) / uScreen.y;
+      `);
+
+    decorate(program);
+
+    const buffers = buildBuffers(xValues, open, high, low, close, bandwidth, lineWidth, numElements);
+    bindBuffers(program, buffers);
```

Comment:
This is what was originally being done with bar but we've changed it to be more consistent with the rest of the code by binding to the buffers when the variable is initially set. I'm assuming we'd want to do that here too?